ArduPlane/quadplane.cpp

```diff
@@ -1561,15 +1562,15 @@ void SLT_Transition::update()
         }
         quadplane.hold_hover(climb_rate_cms);
 
-        if (!quadplane.tiltrotor.is_vectored()) {
+        if (!quadplane.tiltrotor.is_vectored() && quadplane.option_is_set(QuadPlane::OPTION::SUPPRESS_YAW_TRANSITION)) {
             // set desired yaw to current yaw in both desired angle
             // and rate request. This reduces wing twist in transition
             // due to multicopter yaw demands. This is disabled when
             // using vectored yaw for tilt-rotors as the yaw control
             // is needed to maintain good control in forward
             // transitions
             quadplane.attitude_control->reset_yaw_target_and_rate();
-            quadplane.attitude_control->rate_bf_yaw_target(0.0);
+            quadplane.attitude_control->rate_bf_yaw_target(rad_to_cd(quadplane.ahrs.get_gyro().z));
```

Comment:
I'm sure @lthall would be happy for us to change this to a native radians method (https://github.com/ArduPilot/ardupilot/pull/30164). Looks like the two cases updated here are the only users. The method immediately calls `cd_to_rad`.

Reply:
Also it looks like the comment on the method in attitude control is wrong.